docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
+description: "Learn about the Android SDK's release channels and choose the right one."
+sidebar_order: 7000
+---
+
+The Android SDK uses the following channels to release SDK updates:
+
+## `Pre-release`
+
+Pre-releases (alpha, beta, RC,…) are for larger and potentially more impactful changes, such as new features or major versions.
+
+## `Latest`
+
+We continuously release major/minor/hotfix versions from the `main` branch. These releases go through all our internal quality gates and are very safe to use. They're intended to be the default for most teams.
+
+## `Stable`
+
+Releases are promoted from `Latest` to `Stable` when they've been used in the field for some time and at some scale. The length of time a release has been out, the level of adoption, and other quality and stability metrics are considered. These releases will be indicated on the releases page (https://github.com/getsentry/sentry-cocoa/releases/) with the `Stable` label.
+
+## FAQ
+
+### Where can I find the latest stable release for any given SDK?
+
+On the releases pages of the SDK's GitHub repo. Stable releases will be indicated with the label `(Stable)` in the name of the release. You can also use the search box in the top right to search for the newest releases.
+
+### Does this mean the release channels are not stable?
+
+No. Stability and performance are the top priorities for every release. The `Stable` release status means that the release has been tested and proven to work in the wild and at scale.
+
+### How often do we mark SDKs as stable?
+
+The target cadence is to provide stable releases every month.
+
+### Should I only use stable releases?
+
+This is a personal choice for you, your team, and your company. As stated earlier, every release is considered stable, but `Stable` releases have been tested longer with the tradeoff of a slightly delayed availability. This is also the reason why there isn't a pre-release for every release.
+
+### How does this relate to the release process and stability of a feature?
+
+This process doesn't affect feature stability and doesn't change the way features are rolled out. For example, a stable SDK release might include a new feature that's marked experimental.
```

Comment:
good point, we could make it more explicit. maybe something along the lines of

```This process doesn't affect the release process and stability of specific SDK or Product features, such as Session Replay. For example, a new feature that is still experimental might be part of a stable SDK release, and it will be opt-in and clearly marked as experimental.```